airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -190,11 +190,12 @@ def __init__(self, service, project_id):
     def run_query(
             self, bql, destination_dataset_table = False,
             write_disposition = 'WRITE_EMPTY',
+            create_disposition='CREATE_IF_NEEDED',
             allow_large_results=False,
             udf_config = False,
             use_legacy_sql=True,
             maximum_billing_tier=None,
-            create_disposition='CREATE_IF_NEEDED',
+            schema_update_options=(),
```

Comment:
nit: `:type schema_update_options: list` but this is a tuple by default

Reply:
Oh, yes, true! I'll change it!